core/src/main/python/wlsdeploy/util/weblogic_helper.py

```diff
@@ -187,6 +189,14 @@ def get_weblogic_home(self, oracle_home):
             else:
                 wl_home = oracle_home + '/wlserver_10.3'
 
+            # if the path is not a valid directory, try to infer it from the pattern
+            if os.path.isdir(oracle_home) and not os.path.isdir(wl_home):
+                dirs = [f for f in os.listdir(oracle_home) if re.match(r'wlserver.*', f)]
+                if len(dirs) > 0:
+                    wl_home = oracle_home + '/' + dirs[0]
+                else:
+                    wl_home = None
```

Comment:
wl_home is already set to None, just above the if-statement.  Please remove the last else-statement, and I think it should be good to go.

Reply:
wls_home is not none: it gets set to oracle_home + '/' + wls_server_10.3 as default from the previous set of checks. The added code checked that the folder 'detected' is valid, and if not does an extra check to try to infer the folder. If that fails then it sets the wl_home to None again.

Reply:
sorry, you are correct.  I missed that.